onnx/test/model_container_refeval_test.py

```diff
@@ -0,0 +1,139 @@
+# Copyright (c) ONNX Project Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import tempfile
+import unittest
+
+import numpy as np
+import numpy.testing as npt
+
+import onnx
+import onnx.helper
+import onnx.model_container
+import onnx.numpy_helper
+import onnx.reference as refeval
```

Comment:
```suggestion
import onnx.reference
```

Reply:
I changed. The reason why I use from ... import or alias is because the code is shorter and faster to type. Everybody uses `import numpy as np` for that reason.

Reply:
That makes sense. np is a conventional alias. I suggest we establish a convention and keep it consistent if there is a need